server/data/vocabularies.json

```diff
@@ -6655,6 +6655,22 @@
         },
         "unique_field": "qcode"
     },
+    {
+		"_id": "original-metadata",
+		"display_name": "original metadata",
+		"type": "manageable",
```

Comment:
Should this type be `unmanageable` considering it is empty (as the values are retrieved from the item itself)?